src/main/java/com/github/draylar/lilTater/common/Blocks.java

```diff
@@ -17,13 +19,17 @@
     static final Block IRRITATED_TATER;
     static final Block DROWNED_TATER;
 
+    public static final BlockEntityType<LilTaterBlockEntity> LIL_TATER_BLOCK_ENTITY;
+
     static
     {
         LIL_TATER = register("lil_tater", new LilTaterBlock(FabricBlockSettings.of(Material.TNT).hardness(0.3f).build()));
         TATER_TOT = register("lil_tater_tot", new TaterTotBlock(FabricBlockSettings.of(Material.TNT).hardness(0.5f).build()));
         COMPRESSED_TATER_TOT = register("compressed_tater_tot", new CompressedTaterTotBlock(FabricBlockSettings.of(Material.TNT).hardness(1f).build()));
         IRRITATED_TATER = register("irritated_tater", new IrritatedTaterBlock(FabricBlockSettings.of(Material.TNT).hardness(1f).build()));
         DROWNED_TATER = register("drowned_tater", new DrownedTaterBlock(FabricBlockSettings.of(Material.TNT).hardness(1f).build()));
+
+        LIL_TATER_BLOCK_ENTITY = Registry.register(Registry.BLOCK_ENTITY, new Identifier("lil-tater", "lil_tater"), BlockEntityType.Builder.create(LilTaterBlockEntity::new).build(null));
```

Comment:
FYI: in 1.14.4, `BlockEntityType.Builder.create` will also need all the tater blocks.